spring-grpc-spring-boot-autoconfigure/pom.xml

```diff
@@ -88,7 +88,11 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
```

Comment:
I believe this needs to move up out of the "test" dependencies area and mark it w/ `<optional>true</optional>` which will result in it being usable in tests as well.

Reply:
Done.